### PR TITLE
20.6.1 rns

### PR DESCRIPTION
**Pull request by jochman**
In progress

### CODE REVIEW

File: content-repo/extra-docs/releases/20.6.1.md [hunk 1]
```diff
@@ -0,0 +1,595 @@
...
+### Carbon Black Enterprise Protection Pack v1.0.1
+#### Integrations
+##### CarbonBlackProtectionV2  
+Added documentation notes
+
+---
+
+### Carbon Black Enterprise Response Pack v1.0.3
+#### Integrations
+0    ##### _Carbon Black Enterprise Response v2_
```
Comment (yaakovi) [inline content-repo/extra-docs/releases/20.6.1.md:116]:
```suggestion
##### _Carbon Black Enterprise Response v2_  
```

File: content-repo/extra-docs/releases/20.6.1.md [hunk 2]
```diff
@@ -0,0 +1,595 @@
...
+##### EWSO365  
+New Integration EWS O365
+
+---
+
+### EWS Pack v1.1.2
+#### Integrations
+##### EWS v2  
+- Fixed a bug in the **test module** which failed on a delegated mailbox.
+- Improved handling of errors raised in the incident fetch flow.
```
Comment (yaakovi) [inline content-repo/extra-docs/releases/20.6.1.md:221]: same pack

File: content-repo/extra-docs/releases/20.6.1.md [hunk 3]
```diff
@@ -0,0 +1,595 @@
+# Cortex XSOAR Content Release Notes for version 20.6.1 (99999)
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.6.1.md:1]: dont forget to update the build number

File: content-repo/extra-docs/releases/20.6.1.md [hunk 4]
```diff
@@ -0,0 +1,595 @@
...
+---
+
+### Cortex XDR - IOC Pack v1.0.1
+#### Integrations
+##### Cortex XDR - IOC  
+Fixed an issue where trying to push a none existed indicator in ***xdr-iocs-push*** raised an error that failed the command.
+
+---
+
+### Deprecated Content Pack v1.2.0
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.6.1.md:207]: i think we agreed this shouldnt be in the rn ?

File: content-repo/extra-docs/releases/20.6.1.md [hunk 5]
```diff
@@ -0,0 +1,595 @@
...
+#### Playbooks
+##### Malware Playbook - Manual
+Deprecated. Use "Malware Investigation - Manual" playbook instead.
+
+---
+
+### EWS Pack v1.1.0
+#### Integrations
+##### EWSO365  
+New Integration EWS O365
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.6.1.md:217]: need to improve this
the `New` prefix should be before the integration name and we should have the integration description here instead

File: content-repo/extra-docs/releases/20.6.1.md [hunk 6]
```diff
@@ -0,0 +1,595 @@
...
+---
+
+### Google Cloud Compute Pack v1.0.1
+#### Playbooks
+##### IP Whitelist - GCP Firewall  
+Playbook to sync a list of IPs with a GCP Firewall. Moved from IPWhitelisting pack.
+
+---
+
+### HIPAA - Breach Notification Pack v1.0.1
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.6.1.md:271]: we already have that pack in the top

File: content-repo/extra-docs/releases/20.6.1.md [hunk 7]
```diff
@@ -0,0 +1,595 @@
...
+#### Integrations
+##### Joe Security  
+Fixed a bug in **joe-analysis-info** command that wasn't returning a DBotScore.Indicator when was given a URL.
+
+---
+
+### Malware Pack v1.1.0
+#### Playbooks
+##### Malware Investigation - Manual  
+New playbook for "Malware Investigation - Manual".
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.6.1.md:330]: same comment as in ews o365 about new entity

File: content-repo/extra-docs/releases/20.6.1.md [hunk 8]
```diff
@@ -0,0 +1,595 @@
...
+#### Integrations
+##### MongoDB  
+Fixed an issue where nested dictionaries containing a datetime object were not parsed properly.
+
+---
+
+### Office 365 Feed Pack v1.1.1
+#### Integrations
+##### Office 365 Feed  
+Updated integration description.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.6.1.md:390]: lets remove this

File: content-repo/extra-docs/releases/20.6.1.md [hunk 9]
```diff
@@ -0,0 +1,595 @@
...
+### VirusTotal Pack v1.0.1
+#### Integrations
+##### VirusTotal  
+Fixed an issue where the **url** command lacked the default url argument.
+
+---
+
+### Whois Pack v1.1.0
+#### Playbooks
+##### New: TIM - Process Domain registrant With Whois
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.6.1.md:571]: lets add descriptions here

File: content-repo/extra-docs/releases/20.6.1.md [hunk 10]
```diff
@@ -0,0 +1,595 @@
...
+#### Integrations
+##### ipinfo  
+Added support fo0.to trust servers with no valid certificate.
+
+---
+
+### okta Pack v1.0.2
+#### Integrations
+##### Okta v2
+- Added command - okta-get-zone
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.6.1.md:593]: lets unify these 3 bullets into one

File: content-repo/extra-docs/releases/20.6.1.md (hunk 9 above)
Comment (glicht) [reply content-repo/extra-docs/releases/20.6.1.md:571]: We released these playbooks last release. Need to check what cbanged here. They are not NEW anyway. @michalgold please check.

File: content-repo/extra-docs/releases/20.6.1.md [hunk 11]
```diff
@@ -0,0 +1,595 @@
...
+Detonate URL using Group-IB TDS Polygon integration.
+
+
+---
+
+### AWS - EC2 Pack v1.1.1
+#### Scripts
+##### AwsEC2GetPublicSGRules
+- Added support for security groups with only one ingress rule.
+- Playbook to sync a list of IPs with an AWS Security Group. Moved from IPWhitelisting pack.
```
[the remark this answers is not in the document]
Comment (jochman) [reply content-repo/extra-docs/releases/20.6.1.md:80]: image

File: content-repo/extra-docs/releases/20.6.1.md [hunk 12]
```diff
@@ -0,0 +1,599 @@
...
+
+---
+
+### Base Pack v1.0.10
+#### Scripts
+##### CommonServerPython  
+- Added support for the CVE indicator class.
+- Add safeget from python dict function.
+- Fixed an issue where the **argToList** function did not behave as expected. This fix breaks backward compatibility.
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.6.1.md:89]: What does this mean: "This fix breaks backward compatibility"?  @yaakovi I think you wrote this.

File: content-repo/extra-docs/releases/20.6.1.md (hunk 4 above)
Comment (jochman) [reply content-repo/extra-docs/releases/20.6.1.md:207]: I think that we should announce the customers about deprecated content. Maybe in another form, but we should do it.

File: content-repo/extra-docs/releases/20.6.1.md [hunk 13]
```diff
@@ -0,0 +1,599 @@
...
+#### Integrations
+##### SlackV2  
+Added stability improvements for long-running execution.
+
+---
+
+### TIM - Processing Pack v1.1.1
+#### Playbooks
+##### TIM - Indicator Auto Processing  
+Added sub playbook for Whois.
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.6.1.md:558]: Was released already last time. Can be removed. @yaakovi looks like we have a bug in the script.

File: content-repo/extra-docs/releases/20.6.1.md (hunk 12 above)
Comment (Itay4) [reply content-repo/extra-docs/releases/20.6.1.md:89]: @glicht before the change:
`argToList(1)` would output `1`
after the change
`argToList(1)` outputs `[1]`

File: content-repo/extra-docs/releases/20.6.1.md (hunk 9 above)
Comment (jochman) [reply content-repo/extra-docs/releases/20.6.1.md:571]: @yaakovi fixed the rns there so the rn generator took it. I'm removing

File: content-repo/extra-docs/releases/20.6.1.md [hunk 14]
```diff
@@ -0,0 +1,559 @@
+# Cortex XSOAR Content Release Notes for version 20.6.1 (99999)
+##### Published on 23 June 2020
+
+---
+
+Welcome to the 20.6.1 Content release for Cortex XSOAR. Starting from 20.6.0 release, we are restructuring our release notes to be based upon **Content Packs**. One of our team's top priorities is making our Content more accessible and understandable for both users and contributors. In this effort, we recently moved our [Content repo](https://github.com/demisto/content) to work in Packs format were there is a clear separation and grouping of Content artifacts. Each Content Pack provides a clear grouping of related Content artifacts used to either implement a use case, implement an integration or provide a clear set of functionality. Our new release notes are structured around Content Packs and you will see related Content artifacts grouped together according to Packs. We hope you will find this new format useful and clear.
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.6.1.md:6]:
```suggestion
Welcome to the 20.6.1 Content release for Cortex XSOAR. Starting from the 20.6.0 release, we restructured our release notes to be based upon **Content Packs**. One of our team's top priorities is making our Content more accessible and understandable for both users and contributors. In this effort, we recently moved our [Content repo](https://github.com/demisto/content) to work in Packs format were there is a clear separation and grouping of Content artifacts. Each Content Pack provides a clear grouping of related Content artifacts used to either implement a use case, implement an integration or provide a clear set of functionality. Our new release notes are structured around Content Packs and you will see related Content artifacts grouped together according to Packs. We hope you will find this new format useful and clear.
```

File: content-repo/extra-docs/releases/20.6.1.md (hunk 3 above)
Comment (jochman) [reply content-repo/extra-docs/releases/20.6.1.md:1]: 👍

File: content-repo/extra-docs/releases/20.6.1.md (hunk 12 above)
Comment (yaakovi) [reply content-repo/extra-docs/releases/20.6.1.md:92]: wasn't me, https://github.com/demisto/content/pull/7472

**Pull request merged**